third_party/llvm/llvm.BUILD

```diff
@@ -1268,7 +1320,7 @@ cc_library(
     ],
 )
 
-cc_library(
+llvm_cc_library(
```

Comment:
Did you consider using `cc_library(copts = LLVM_COPTS, ...)` instead of a `llvm_cc_library`?  Defining a new function for this seems more powerful that necessary.